lib/parallel_tests/tasks.rb

```diff
@@ -56,10 +56,10 @@ def check_for_pending_migrations
         end
       end
 
-      # parallel:spec[:count, :pattern, :options]
+      # parallel:spec[:count, :pattern, :options, :serialize_stdout]
```

Comment:
sounds better ... but also kind of weird that it passes commandline options
through a rake task into an executable ... when it could just call the
executable ...

confusing: rake parallel:test[2,'','', '--foobar']
clean: parallel_test -n2 --foobar

On Sun, May 8, 2016 at 11:44 PM, Efesto <notifications@github.com> wrote:

> In lib/parallel_tests/tasks.rb
> <https://github.com/grosser/parallel_tests/pull/503#discussion_r62455103>:
>
> > @@ -56,10 +56,10 @@ def check_for_pending_migrations
> >          end
> >        end
> >
> > -      # parallel:spec[:count, :pattern, :options]
> > +      # parallel:spec[:count, :pattern, :options, :serialize_stdout]
>
> I thought it could be a good idea because I found the rake task to be
> useful but not as much power as the executable but I recognize that
> serialize_stdout is not as much important like the others.
>
> Maybe the 4th parameter could be a more generic options parameter which is
> passed directly to the executable?
> If so the current options could be renamed :test_options.
>
> What do you think about it?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/parallel_tests/pull/503/files/0ae9a7382d7d50a06fb114bf8dee64e96be4520d#r62455103>
>